assets/sass/local.sass

```diff
@@ -292,6 +292,9 @@ dt.summary-list__key__wider
     width: 150px
     margin-bottom: 32px
 
+.date-picker-container span
+    width: 850px
```

Comment:
Out of curiosity, what's the reason for this?

Reply:
I am not sure, a lot of this wasn't done by me. I'll have a look at this in a follow up..